tensorflow_datasets/core/download/download_manager.py

```diff
@@ -336,6 +339,10 @@ def _handle_download_result(
           url=resource.url, url_path=url_path, url_info=url_info)
     elif resource.url not in self._url_infos:
       if self._force_checksums_validation:
+        if tf.io.gfile.exists(self._checksums_path):
+          url_infos = checksums._get_url_infos(self._checksums_path)  # pylint: disable=protected-access
```

Comment:
`_get_url_infos` should be renamed `get_url_infos` if it is now part of the public API.